### ISSUE TITLE
프로젝트 리뷰

### ISSUE THREAD

**Issue by crongro** [codesquad-member-2020/signup-6#21]
issue close할때 commit 번호를 같이 추가해서 기록을 남겨보세요
서비스를 유지할 건 아니지만 습관적으로 연습하는 것이죠.

https://github.com/codesquad-memeber-2020/signup-6/issues/10
이렇게 문서정리를 하면 어디에다가 했는지도 남기면 좋을 듯.
문서는 wiki를 적극 활용해보세요.

아직 며칠 안됐지만 PR이 ios만 보이네요. 기능단위로 개발하시면서 바로바로 PR & 머지하면 좋겠어요.

처음치고 잘 하고 계십니다!
모두 힘내시고요!

**Comment by reesekimm** [comment]
@crongro 

> 기능단위로 개발하시면서 바로바로 PR & 머지

기능단위 개발 하고 PR 없이 바로 dev 브랜치에 머지해왔습니다^^;
PR을 거쳐서 머지할때의 좋은 점은 **해당 기능 개발이 완료되었음을, 혹은 이슈가 해결되었음을 명시적(/공식적)으로 팀원들과 공유할 수 있기 때문**이라고 이해해도 될까요?

**Comment by crongro** [comment]
> PR을 거쳐서 머지할때의 좋은 점은 **해당 기능 개발이 완료되었음을, 혹은 이슈가 해결되었음을 명시적(/공식적)으로 팀원들과 공유할 수 있기 때문**이라고 이해해도 될까요?

네 맞아요. 서로 코드를 리뷰하고, 확인절차후에 중요한 브랜치에 머지를 결정짓죠. 그렇지 않은 경우도 많긴해요. 그냥 바로바로 머지고고 ㅎㅎ

**Comment by crongro** [comment]
- 브랜치관리를 잘 해서 좋네요.
- 이슈 닫을때 커밋번호 연결까지 하시는 것도  참 좋네요. 
- label은 상태를 표현하진 않는게 일반적인데 completed 가 있어서 의아했음.
- 이슈관리 하고 상태를 업데이트 하는 건 모두다 같이 하는 프로젝트에서 필요한 것이죠. 어렵지만  서로의 상태를 잘 공유하고 기록을 남겨두고(issue등에)하는 것이 그래서 필요해요!

**Comment by reesekimm** [comment]
> * label은 상태를 표현하진 않는게 일반적인데 completed 가 있어서 의아했음.

커밋 번호를 연결해서 이슈를 닫는게 default 브랜치(dev)에서만 가능해서,
각자 feature 브랜치에서 작업 완료했을 경우 `complete` 라벨을 붙여 완료 상태를 공유했습니다:)

**Comment by honux77** [comment]
크롱 말대로 GitHub의 PR은 여러가지로 강력한 기능을 가지고 있기 때문에 꼭 쓰는게 좋습니다. 저는 개인 프로젝트에 셀프 리뷰를 할 때도 PR을 써요.  커밋은 원자적으로 PR은 기능단위로 하고, 지금은 사용하지 않겠지만 현업에서 revert를 할 때도 PR 단위 revert가 종종 있습니다. 이럴 때 GitHub을 사용하면 쉽게 됩니다.

**Comment by honux77** [comment]
전반적으로 프로젝트 관리는 잘 되고 있는 것 같아요. 아직 배포는 안 보이지만 모두들 수고하셨어요.
